.github/workflows/on-pull-request.yaml

```diff
@@ -34,8 +34,5 @@ jobs:
     - uses: actions/checkout@v4
     - name: Build Sphinx Image
       run: |
-        source <(curl -sL http://ci.q-ctrl.com)
-        ./ci docker build \
-          -f ./docs/Dockerfile \
-          --suffix -docs
```

Comment:
@jrosco What does this do and are we going to miss these args?

Reply:
@aggronerd only builds image, so the `--suffix -docs` seems pointless